### ISSUE TITLE
Move field displays from bottom to top?

### ISSUE THREAD

**Issue by jakabo27** [rejuvenate/lovelace-horizon-card#111]
Hello, is there any way to modify where a field is in the card?  I only really want the fields Sunrise/Sunset/SolarNoon (see image below), but would love for them to all be on the same line instead of solar noon on the bottom.  Is there any way to change this, maybe with card-mod?  

![image](https://github.com/rejuvenate/lovelace-horizon-card/assets/29758597/f5a2334c-0bd6-4660-adc6-c1291f8c8227)

**Comment by DM-AU** [comment]
Yes please! Dashboards space is very limited and moving solar noon between sunset and sunrise would be ideal.

**Comment by Hannah-Bee** [comment]
I wanted to request the same feature. I don't even need the noon label. It would be enough for me if the noon time was written in smaller font size between sunrise and sunset.
In my case, the reason is also that my dashboard is relatively full and this adjustment would make the card much more compact.
